packages/suite/src/reducers/suite/desktopUpdateReducer.ts

```diff
@@ -0,0 +1,51 @@
+import produce from 'immer';
+import { DESKTOP_UPDATE } from '@suite-actions/constants';
+import { Action } from '@suite-types';
+import { UpdateInfo, UpdateProgress, UpdateWindow } from '@suite-types/desktop';
+
+export interface State {
+    state?: 'checking' | 'available' | 'not-available' | 'downloading' | 'ready';
```

Comment:
nitpick: maybe better to make state property required with some 'initial' initial value. It then helps that you don't need this elsewhere: 

```
!['checking', 'not-available'].includes(desktopUpdate.state || '')
```